lib/kubernetes-deploy/runner.rb

```diff
@@ -131,6 +138,8 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       success = false
     ensure
       @logger.print_summary(success)
+      tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']})
```

Comment:
Actually, what is the value of the SHA tag? Wouldn't the idea of emitting metrics like this be to look over several deploys? (the duration info is already printed in the deploy output if you're talking about a single one).